cronjobs/FileLoader.py

```diff
@@ -0,0 +1,128 @@
+import os, sys
+sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
+
+from models import ScheduleHistory, CallingPointHistory
+from darwinpush.messagefactories.xml import ScheduleXMLMessageFactory
+import darwinpush.xb.pushport as pp
+
+import re
+
+class ScheduleFileLoader:
+
+    def __init__(self, file_location):
+        assert(file_location)
+        self.file_location = file_location
+
+    def update_daily_schedules(self):
+
+        collect_mode=False
+        journey_buffer = []
+
+        for line in open(self.file_location):
+
+            if '<Journey' in line:
+                assert(not collect_mode)
+                collect_mode=True
+
+            elif '</Journey' in line:
+                assert(collect_mode)
+
+                journey_buffer.append(line)
+                journey_string = "".join(journey_buffer)
```

Comment:
I don't understand the solution proposed. So once we find the closing block, we would delete it from the string and re-consider the line?